### PR TITLE
Graph annotation - Add Node to a Group

### PR DESCRIPTION
**Pull request by ramramps**
This PR fixes the below issues:

**MAGN-7177 Tool tip for view change still shows Ctrl + G as a shortcut, it should be Ctrl +B**
     Updated the tooltip in Resources file

**MAGN-6978 Group titles need a little bit more room where they start**
      Changed the boundary calculation in Annotation Model

**MAGN-7124 As a user, I want to add a note, node or group to an existing group**
     This is the major change in this PR. To add a node or note to a group, select any group and right click on any node / note - click AddToGroup on the  context menu. 
 
- [x] @pboyer

### CODE REVIEW

File: src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs [hunk 1]
```diff
@@ -944,9 +946,34 @@ internal void UngroupModel(object parameters)
...
         internal bool CanUngroupModel(object parameter)
         {
+            var tt = DynamoSelection.Instance.Selection.OfType<ModelBase>().Any();
             return DynamoSelection.Instance.Selection.OfType<ModelBase>().Any();
         }
 
+        internal bool CanAddModelsToGroup(object obj)
+        {
+            var tt = DynamoSelection.Instance.Selection.OfType<AnnotationModel>().Any();
+            return DynamoSelection.Instance.Selection.OfType<AnnotationModel>().Any();
```
Comment (pboyer) [inline src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs]: Unused `tt` variables in both of these methods.

File: src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs [hunk 2]
```diff
@@ -944,9 +946,34 @@ internal void UngroupModel(object parameters)
...
+
+        internal void AddModelsToGroup(object parameters)
+        {
+            if (null != parameters)
+            {
+                var message = "Internal error, argument must be null";
+                throw new ArgumentException(message, "parameters");
+            }
+            //Check for multiple groups - Delete the group and not the nodes.
+            foreach (var modelb in DynamoSelection.Instance.Selection.OfType<ModelBase>().ToList())
```
Comment (pboyer) [inline src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs]: No need for the `ToList()` here.

File: src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs [hunk 3]
```diff
@@ -98,10 +100,20 @@ private void AnnotationView_OnMouseLeftButtonDown(object sender, MouseButtonEven
             {
                 var annotationGuid = this.ViewModel.AnnotationModel.GUID;
                 ViewModel.WorkspaceViewModel.DynamoViewModel.ExecuteCommand(
-                    new DynCmd.SelectModelCommand(annotationGuid, Dynamo.Utilities.ModifierKeys.Shift));
+                    new DynCmd.SelectModelCommand(annotationGuid, Keyboard.Modifiers.AsDynamoType()));
+
+                //ViewModel.WorkspaceViewModel.DynamoViewModel.AddModelsToGroupModelCommand.Execute(null);
```
Comment (pboyer) [inline src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs]: Let's just kill dead code unless its meaningful.

**Comment by pboyer** [comment]
Hey @ramramps, 

This looks solid.  Most of these issues are surface defects - let's fix them.  Can you update the related group testing task on YT (i.e. write automated tests) to ensure that MAGN-7124 gets covered?

~Peter

**Comment by ramramps** [comment]
@pboyer - Thanks peter. Fixed those surface defects.
I have unit test cases around every function. I have to write some regression tests as well. Hopefully I should have them for review tomorrow.

**Pull request merged**